drivers/gpio/gpio_tca6424a.c

```diff
@@ -0,0 +1,572 @@
+/*
+ * Copyright (c) 2022 Chromium OS Authors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#define DT_DRV_COMPAT ti_tca6424a
+
+
+#include <zephyr/drivers/gpio/gpio_utils.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/sys/util.h>
+LOG_MODULE_REGISTER(gpio_tca6424a, CONFIG_GPIO_LOG_LEVEL);
+
+/* TCA6424A auto increment register addresses */
+#define TCA6424A_REG_INPUT_PORT		     0x80
+#define TCA6424A_REG_OUTPUT_PORT	     0x84
+#define TCA6424A_REG_POLARITY_INVERSION_PORT 0x88
+#define TCA6424A_REG_CONFIGURATION_PORT	     0x8C
```

Comment:
Think you can drop the `_PORT` suffix. Could understand if you were using indivudual register for say `_PORT_0` etc... but since you are defining the first one, just call them `TCA6424A_REG_INPUT` `TCA6424A_REG_OUTPUT` etc.

Reply:
Done